ui/app/components/sidebar/user-menu.hbs

```diff
@@ -54,6 +54,7 @@
                 @container="#container"
                 class="in-dropdown link is-flex-start"
                 {{on "click" (fn (set-flash-message "Token copied!"))}}
+                data-test-clipboard-value={{this.auth.currentToken}}
```

Comment:
nitpicking, but it's odd to see so many variations of the same entity/name: `data-test-copy-button`, `data-test-clipboard-text`, `data-test-tooltip-copy`, `data-test-clipboard-value` 😐

Reply:
Yep - I iteratively replaced `data-clipboard-text` as I went through the tests - and purposefully differentiated the selector names to track which ones were updated. 

We don't have a consistent way of adding test selectors on our team, though we've _started_ moving towards using a general selector file so we can reuse selectors